en/Content Moderation Practices

```diff
@@ -0,0 +1,39 @@
+
+Where we receive duplicative reports about the same piece of content, we will do a single review and will notify later reporters that the content has already been assessed, rather than reviewing and responding separately to each report.
+
+## Moderation decisions
+When a moderator determines that content does not violate our policies, we will notify the reporter of that determination and will take no action against the content.  In most cases, we will provide the reporter the opportunity to appeal.
+
+When a moderator finds that one of our policies has been violated, they will take the action dictated by that policy, including: 
+* adding a content warning, 
+* removing the content, or 
+* suspending the account.  
+We will notify both the reporter and the user that a violation was found, and we will provide the user the opportunity to appeal.  
+
+Where a violation is punishable by a warning or content removal, additional violations may lead to suspension.  In such cases, we will also notify the user that their account is being suspended based on a history of multiple violations.  We will provide opportunities to separately appeal both the latest moderation decision, and the resulting suspension.  In some products, these appeals can be combined, while in others they must be done separately. 
+
+## Appeals
+We accept two kinds of moderation-related appeals: 
+* from reporters whose reports are not actioned; and 
+* from users whose content is removed or whose accounts are penalized for violating our content policies.  
+In both cases, we provide a form where the party can submit their appeal and explain why they believe the prior decision was incorrect, and did not align with our policies or with the applicable legal requirements.  These appeals rely on the same case management tool that we use in the first stage of our moderation process - which helps us to receive and track appeals, and to review the policy, the initial report, and any other information that was used in making our moderation decision.
+
+Appeals will be routed to a dedicated queue upon receipt.  Depending on the type of violation involved, items in that queue will be reviewed by other members of the moderation team, the applicable product team, or our Mozilla-wide Trust & Safety or Legal Teams.  Appellate reviewers will review the applicable policy or policies and any relevant internal guidelines, as well as the information submitted by the appellant, to determine whether the original decision was appropriate and in line with Mozilla’s rules.
+
+When a user prevails in their appeal, we will inform the user, and the prior action against their content or their account will be reversed.  When a reporter prevails in their appeal, we will inform both the user and the reporter, and will action the violative content as dictated by the applicable policy.
```

Comment:
Double check the last sentence `... and will action the violative content as ...` it doesn't sound right, missing a word or incorrect use of a word.

Reply:
Based on the rest of the content, this seems correct (although very legalese).

action = take actions against, i.e. "we will take actions against a content that violates rules"

Reply:
The use of "action" in this doc is a verb, not a noun. We can use suggested text.